pkg/controller/mpi_job_controller.go

```diff
@@ -134,6 +134,11 @@ var (
 		Name: "mpi_operator_job_info",
 		Help: "Information about MPIJob",
 	}, []string{"launcher", "namespace"})
+	mpiJobsDurationHistogram = promauto.NewHistogramVec(prometheus.HistogramOpts{
+		Name:    "mpi_operator_jobs_duration_seconds",
+		Help:    "Duration of MPI jobs in seconds",
+		Buckets: prometheus.LinearBuckets(0, 10, 60),
```

Comment:
60 buckets of 10 seconds, for a total of 10min? Are you sure this is what's most useful?
In any case, leave a comment for the max bucket.